Bottom: a55aa671e92ec72e0cb63694778b43539464ff02
Top:    2bc6cb096178e372d949d4049bbad45f93a2d518
Author: Peter Clifton <pcjc2@cam.ac.uk>
Date:   2011-01-25 21:25:49 +0000

Refresh of fsaa-test

---

diff --git a/src/hid/gtk/gtkhid-gl.c b/src/hid/gtk/gtkhid-gl.c
index 0e71e72..dd11c21 100644
--- a/src/hid/gtk/gtkhid-gl.c
+++ b/src/hid/gtk/gtkhid-gl.c
@@ -1040,6 +1040,8 @@ ghid_start_drawing (GHidPort *port)
   if (!gdk_gl_drawable_gl_begin (pGlDrawable, pGlContext))
     return FALSE;
 
+  glEnable (GL_MULTISAMPLE_ARB);
+
   return TRUE;
 }
 
@@ -1049,6 +1051,8 @@ ghid_end_drawing (GHidPort *port)
   GtkWidget *widget = port->drawing_area;
   GdkGLDrawable *pGlDrawable = gtk_widget_get_gl_drawable (widget);
 
+  glDisable (GL_MULTISAMPLE_ARB);
+
   if (gdk_gl_drawable_is_double_buffered (pGlDrawable))
     gdk_gl_drawable_swap_buffers (pGlDrawable);
   else
